docs/components/toast/elements.md

```diff
@@ -15,7 +15,7 @@ Once you have imported the elements package, import the toast api package.
 
 
 ```js
-import { toast, removeToast, updateToast } from '@warp-ds/elements/toast';
+import { toast, removeToast, updateToast } from '@warp-ds/elements';
```

Comment:
Is it something we've changed? It is like the first example in the fabric docs https://elements.fabric-ds.io/toast.html
`import { toast, removeToast, updateToast } from '@warp-ds/elements/toast`

Reply:
No we haven't changed it. But when I tested importing it myself that was how it was done, so I guess it may have been wrong in fabric as well